frontend/docs/seo-guide.md

```diff
@@ -0,0 +1,393 @@
+
+#### Redirections
+
+A redirect is a way to send both users and search engines to a different URL from the one they originally requested. The three most commonly used redirects are 301, 302, and Meta Refresh.
+
+It is important to highlight some facts about them: 
+
+- **All redirects carry a degree of SEO risk**.
+- While 3xx redirects preserve PageRank, 301s remain the preferred method of choice for permanent redirects.
+- Beyond PageRank, all other rules about redirection remain. If you redirect to a non-relevant page, or buy a website in order to redirect 1,000 pages to your homepage, you likely won’t see much of a boost.
+- Successful migrations to HTTPS are now less prone to lose PageRank.
+- **The best redirect is where every other element stays the same, as much as possible, except for the URL.**
+
+> What means each redirection? [Read more...](https://moz.com/learn/seo/redirection)
+
+#### Robots Exclusion Protocol (REP)
+
+The Robots Exclusion Protocol (REP) is a conglomerate of standards that regulate Web robot behavior and search engine indexing. Despite the "Exclusion" in its name, the REP covers mechanisms for inclusion too.
+
+Some of this mechanisms are: robots.txt, rebots meta elements, microformat relationship-nofollow and sitmaps(We talked about it a few line up). We can divide them in two groups:
+
+- **Crawler directives:** *(robots.txt, sitemaps)* Suggest to crawlers what they should crawl and must not crawl. 
+
+
+- **Indexer directives:** *(REP tags, microformats)* Require crawling.
+
+There’s an important but subtle difference between using **meta robots** and using **robots.txt** to prevent indexation of a page. Using meta robots "noindex,follow" allows the link equity going to that page to flow out to the pages it links to. If you block the page with robots.txt, you’re just flushing that down the toilet.
+
+_Why:_
+
+> This is a very complex topic, so I recommend you to [read more...](https://developers.google.com/analytics/solutions/)
+
+### 2.6 Social networks
+
+#### Social sharing buttons
+
+Social networks may not play direct role in ranking your site. But social shares generate more eyes focus on your content, and more people you attract, the more likely someone is to link. 
+
+_Why:_
+> A study by [BrigthEdge](https://www.brightedge.com/) found that prominent social sharing buttons can increase social sharing by 700%. [Read more...](https://blog.hubspot.com/blog/tabid/6307/bid/24994/using-social-sharing-buttons-leads-to-7x-more-mentions-data.aspx)
+
+#### The Open Graph
+
+The Open Graph protocol enables any web page to become a rich object in a social graph. For instance, this is used on Facebook to allow any web page to have the same functionality as any other object on Facebook.
```

Comment:
used by Facebook

Reply:
object in Facebook

Reply:
I think the correct way is really  "...object from Facebook".